### ISSUE TITLE
title 

### ISSUE THREAD

**Issue by muskanrawal** [stefanbund/cis1-a#12]
'The Efficiency of Self-Nurture in Coping with Stress from Change, in Community College Student Populations' .. does this title change if we choose a different stressor with a different solution (pressure behavioral) ?

**Comment by stefanbund** [comment]
Yes, my title is simply an example. You must have your own, original title. But I recommend using the same kind of title that I used. Also, please keep your title very specific to your results, and limited in its claims.

Stefan
